src/config.ts

```diff
@@ -42,6 +42,7 @@ export default {
       'superman-superwoman',
       'simple',
       'just',
+      'fellow',
```

Comment:
This is a duplicate from line `23` https://github.com/EddieHubCommunity/EddieBot/pull/620/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R23